sites/fast-website/src/public/index.html

```diff
@@ -3,12 +3,114 @@
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
```

Comment:
For future reference, we may want to use webpack loaders so we can do some processing once the stylesheets start to increase in complexity

Reply:
I am good with that.